config/config.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ConfigMap 
```

Comment:
Shall we define a new API in federation for Controller-Manager configuration? validation and defaulting would be difficult in the current form (as a configmap).

Reply:
I prefer to have new API for configuration management. I add comments in issue ticket for discussion: https://github.com/kubernetes-sigs/federation-v2/issues/114#issuecomment-475083315